src/components/Header/DashBoard/index.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image';
+import styles from './DashBoard.module.scss';
+import MenuIcon from '/public/svgs/menu.svg';
+import SettngIcon from '/public/svgs/setting.svg';
+import InviteIcon from '/public/svgs/invite.svg';
+
+function DashBoard() {
+  return (
+    <>
+      <div className={styles.container}>
+        <div className={styles.box}>
+          <Image className={styles.menu} src={MenuIcon} alt="menu" />
```

Comment:
```suggestion
          <Image className={styles.menu} src={MenuIcon} alt="menu" />
```

` <Image className={styles.menu} src='/svgs/menu.svg' alt="menu" />`
이렇게 사용하면 상위에 있는 import 문을 줄일 수 있을 것 같네요!